WorkloadManagementSystem/PilotAgent/PilotLogger/PilotLogger.py

```diff
@@ -0,0 +1,211 @@
+""" Pilot logger module for the remote loggin system
+"""
+
+__RCSID__ = "$Id$"
+
+import pika
+import sys
+from PilotLoggerTools import (
+    generateDict,
+    encodeMessage,
+    generateTimeStamp,
+    isMessageFormatCorrect
+    )
```

Comment:
you don't need parenthesis here. Style guide: https://www.python.org/dev/peps/pep-0008/#imports